presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -41,6 +41,7 @@
     private int maxPartitionsPerWriter = 100;
     private List<String> hadoopConfigResources = ImmutableList.of();
     private double minimumAssignedSplitWeight = 0.05;
+    private boolean parquetDereferencePushdownEnabled = false;
```

Comment:
Thanks for the update! Could you change it to `true`? Otherwise, there'll be a checkstyle error.

Reply:
I changed it to `private boolean parquetDereferencePushdownEnabled`, in this way checkstyle should be fine